pkg/sources/main.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/eagraf/habitat/pkg/sources/sources"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var reader sources.Reader
+var writer sources.Writer
+
+var rootCmd = &cobra.Command{
```

Comment:
i was mostly just using this for testing, i think we actually want to expose an a server / API for sources. I'll add that in a later commit and probably move code into `habitat/cmd` that calls the API rather than calling the functions directly. What do you think?